src/test/components/SourceView.test.js

```diff
@@ -0,0 +1,101 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// @flow
+
+import React from 'react';
+import { Provider } from 'react-redux';
+import { stripIndent } from 'common-tags';
+
+import { ProfileViewer } from 'firefox-profiler/components/app/ProfileViewer';
+import { updateUrlState } from 'firefox-profiler/actions/app';
+import { viewProfile } from 'firefox-profiler/actions/receive-profile';
+import { stateFromLocation } from 'firefox-profiler/app-logic/url-handling';
+import { ensureExists } from 'firefox-profiler/utils/flow';
+
+import {
+  render,
+  screen,
+  within,
+} from 'firefox-profiler/test/fixtures/testing-library';
+import { blankStore } from 'firefox-profiler/test/fixtures/stores';
+import { getProfileFromTextSamples } from 'firefox-profiler/test/fixtures/profiles/processed-profile';
+import { fireFullClick } from 'firefox-profiler/test/fixtures/utils';
+
+// We're not interested in the timeline in this test
+jest.mock('../../components/timeline', () => ({
+  Timeline: 'custom-timeline',
+}));
```

Comment:
I personally think module mocking is terrible, but I'm not the one maintaining this test, so I'm fine with this.